### DISCUSSION TITLE
Hydratation returns a 404 until I edit a file, any idea why?

### DISCUSSION THREAD

**Discussion by RSickenberg** [nuxt/nuxt#27264] [Questions]
https://github.com/nuxt/nuxt/assets/16922548/8fe896d0-a0ec-47f1-ab7a-e2ed07678e50

I searched for:
- HMR issues,
- Vue Vite configs,
- Any nginx config mismatches,
- [This issue](https://github.com/nuxt/nuxt/issues/12003)
- ...

But, in local without passing from a docker node container it works better (except the CORS).
My `Dockerfile` and `entrypoint` both generate a `npm run build`.

My take is, it's an issue with `hmr` but I'm not sure.

Note, I'm more a PHP / Backend developer so I might be around the scope and missed something big.

**Comment by Mortos98** [comment]
also just hit this, did you manage to get anywhere with this?

**Reply by RSickenberg** [reply]
Make sure the cash isn't for something. I found a way; switching browsers showed me the way.
